### PR TITLE
Add lint check and reformat Protocol Buffer definitions

### PR DESCRIPTION
**Pull request by gino-m**

### CODE REVIEW

**Comment by gino-m** [comment]
@sufyanAbbasi PTAL?

File: .github/workflows/test-pr.yml [hunk 1]
```diff
@@ -74,6 +74,25 @@ jobs:
...
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "18"
+
+      - name: Run buf linter
```
Comment (sufyanAbbasi) [inline .github/workflows/test-pr.yml:93]: nit: spell it out for the people in the back, e.g. `Run buf (protobuf) linter`

**Review by sufyanAbbasi — APPROVED** [review]
Great work, thank you for following up on this!

**Review by sufyanAbbasi — APPROVED** [review]
Thank you!

**Pull request merged**